btrack/napari/main.py

```diff
@@ -135,6 +135,10 @@ def run(btrack_widget: Container, configs: TrackerConfigs) -> None:
         properties=properties,
         graph=graph,
         name=f"{segmentation}_btrack",
+        scale=segmentation.scale if hasattr(segmentation, "scale") else None,
+        translate=segmentation.translate
+        if hasattr(segmentation, "translate")
+        else None,
```

Comment:
This isn't particularly satisfying but I guess it's a good compromise to use the scaling if it's there and don't if it isn't (like in the tests). What do you think?

Reply:
Where do these properties come from? They seem like layer properties rather than those of the data...

Reply:
Not sure. @p-j-smith can you advise? These were needed to play nicely with our `epitools` work.

Reply:
> They seem like (napari) layer properties

yep exactly, `segmentation` here is an `Image` layer. The issue was with the test - in the viewer, if you add a new image layer it will be added to the dropdown box automatically. However, for some reason when adding the layer programatically (as in the test), the layer isn't added to the dropdown box. This meant the value returned was `None` rather than an `Image` layer. So explicitly adding the layer to the dropdown box fixes the test